FileSaver/FileSaver.d.ts

```diff
@@ -30,4 +30,8 @@ interface FileSaver {
     ): void
 }
 
-declare var saveAs: FileSaver;
```

Comment:
The package documentation in https://github.com/eligrey/FileSaver.js refers to a global `saveAs`.

```ts
var blob = new Blob(["Hello, world!"], {type: "text/plain;charset=utf-8"});
saveAs(blob, "hello world.txt");
```

when used as a module however, `saveAs` would be a property on the module, see https://github.com/eligrey/FileSaver.js/blob/master/FileSaver.js#L183

so it should be:

```ts
declare var saveAs: FileSaver;

declare module "file-saver" {
    var fileSaver: { saveAs: typeof saveAs };
    export = fileSaver
}
```